web/src/main/java/se/inera/intyg/rehabstod/service/sjukfall/GetSickLeaveSummaryServiceImpl.java

```diff
@@ -58,7 +59,8 @@ public SickLeaveSummary get() {
                 null,
                 null
             ),
-            INCLUDE_PARAMETERS
+            INCLUDE_PARAMETERS,
+            SHOULD_PDL_LOG
```

Comment:
Misleading name. Maybe call it something like SHOULD_NOT_PDL_LOG. 

Could be good to clean up the name of the other constant (INCLUDE_PARAMETERS).